src/AutorestSwift/View Models/ObjectViewModel.swift

```diff
@@ -33,20 +33,79 @@ import Foundation
 struct ObjectViewModel {
     let name: String
     let comment: ViewModelComment
-    let objectType = "struct"
     let properties: [PropertyViewModel]
-    let isErrorType: Bool
+    let constants: [ConstantViewModel]
+    let hasConstants: Bool
+
+    var inheritance = "NSObject"
```

Comment:
Why we need to inherit from "NSObject", can this not specify?

Reply:
Right, I debated what to put here. In this case NSObject is a legitimate value, but would essentially be a sentinel that something went wrong since it should be "Codable" with, possible, "Swift.Error". I spent a lot of time trying to figure out what was happening with a struct that was showing only "Codable" because it was the default. A sentinel would have saved a lot of troubleshooting.